tasks/config.yml

```diff
@@ -88,3 +88,27 @@
      src: bbb/bigbluebutton.j2
      dest: /etc/nginx/sites-available/bigbluebutton
   notify: reload nginx
+
+- name: ensure default recording is disabled
+  lineinfile:
+    path: /usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties
+    regexp: '^disableRecordingDefault'
+    line: 'disableRecordingDefault=true'
+  when: bbb_disable_recordings
```

Comment:
Thanks. Could you maybe change this, so it is possible to re-enable these options by setting the variable to true?

Reply:
I would say: when `bbb_disable_recordings` is false, all these options should be set to their default values.